### PR TITLE
Pull request for "Using Bootstrap" project

### PR DESCRIPTION
**Pull request by jose-Abel**
Hello,

This is the Pull Request for the 7th project on HTML and CSS "Using Bootstrap" where we clone the webpage of newsweek.com applying the Bootstrap framework for the responsiveness of the site most of it, and some general padding and margins where the framework was really good at.

Hopping to get amazing feedback!

Happy coding!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,691 @@
...
+  width: 100%;
+}
+
+.navbar-news {
+  color: #9b8583;
+  font-size: 1.5rem;
+}
+
+.news-nav-a-tag:hover {
+  color: #f72210 !important;
```
Comment (MuriloRoque) [inline style.css]: Please, don't use !important.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 
## Hello @jose-Abel and @bmuhamood, you guys did an amazing job building this website! Here are my pros:
- Everything is in place, just like the original page;
- The page is responsive :clap: :clap: :clap: 
- The live link works properly;
- You used Bootstrap correctly, your code looks nice!
- Good commit, PR description, and README file :partying_face: 

## However, there are still somethings we need to change in order to make it look perfect:

# General

- [x] Please, use semantic HTML tags (i.e.: < section > and/or < article >) instead of only divs and spans.

- [x] For your CSS, please check my comment in your code below, it's only one little detail, the rest is great!

# Page

- [x] Please, make your borders lighter, the original is different:

## Yours:
Screenshot from 2020-08-13 17-58-20
## Original:
Screenshot from 2020-08-13 17-58-17
<br>
- [x] In your bottom footer, you only added the first link, the others are missing:
Screenshot from 2020-08-13 17-55-11
<br>

- [x] Your page has a horizontal scroll in all screen sizes, please fix that:
Screenshot from 2020-08-13 18-11-57
<br>

- [x] The original page has the location to the left of the navbar, while yours doesn't:

## Yours
Screenshot from 2020-08-13 18-13-43


## Original:
Screenshot from 2020-08-13 18-13-40
<br>

- [x] The main three columns section is aligned with the other sections both at the top and at the bottom. Currently, your page is only aligned at the top, check the bottom:

## Yours:
Screenshot from 2020-08-13 18-16-24
## Original:
Screenshot from 2020-08-13 16-33-08

- [x] For the Tablet size, please check this section below 'Opinion' and above 'In the magazine'. Also check your 'Editor's Pick' section for the same problem. The images are too big, it looks like this was supposed to happen only on small screens :thinking: :

## Yours:
Screenshot from 2020-08-13 18-22-18
## Original:
Screenshot from 2020-08-13 18-22-22

# Summary

- Your project looks good, it's ALMOST identical to the original, you just need to work a little more and you will be done in no time :muscle: 
- Your communication skills are great, good project structure, commit messages, PR and README file :partying_face: 


## Kindly make the changes and ask for another review :pray: 

## If you guys have any doubts, please talk to me on Slack :handshake:

File: style.css (hunk 1 above)
Comment (jose-Abel) [reply style.css]: Hello, thanks for the feedback, we were able to improve our project thanks to it, best regards!

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello again @jose-Abel and @bmuhamood :wave: 

## Thank you very much for making the changes I requested, you are awesome :clap: :clap: :clap: 

## You may merge to master! Good luck in your next projects, wish you the best! :tada: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**